### PR TITLE
Update ReadMe

### LINKED ISSUE #24: README for repo does not match RandomGifBot
The README still has the generic echo bot description.

We can add a short description and screenshots for the @Gifbot :)

### PR DESCRIPTION
**Pull request by alancast**
Add contributing steps
Add MIT License

Resolves #24 and #25

### CODE REVIEW

File: Contributing.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+- Create the `.bot` file. Either download the `.bot` file from our Azure resource if you have access, or use the one from your created bot above
+- Run `npm install` in the root of the bot project
+- Finally run `npm run build` and then `npm start`
+
+### Testing the bot using Bot Framework Emulator
+
+[Microsoft Bot Framework Emulator](https://github.com/microsoft/botframework-emulator) is a desktop application that allows bot developers to test and debug their bots on localhost or running remotely through a tunnel.
+
+- Install the Bot Framework Emulator from [here](https://aka.ms/botframework-emulator)
```
Comment (xinyi-joffre) [inline Contributing.md]: Should we mention that the emulator does not work with the Delete button?

Maybe add link to how to side load app?

File: README.md [hunk 2]
```diff
@@ -1,71 +1,29 @@
-# RandomGifBot
+# GifBot
 
-A cute little bot friend who gives your team random GIFs of whatever you ask for
+GifBot is your cute little bot friend who gives your team random GIFs. GifBot is developed in TypeScript, hosted on Azure, and uses Giphy as it's back end. It can be connected to a variety of things, but it's main use case is Microsoft Teams. This is also the app it is most thoroughly tested in (not a coincidence).
```
Comment (xinyi-joffre) [inline README.md]: "its back end" "its main use case"

Comment (alancast) [reply README.md]: d'oh :-( fixed now

File: Contributing.md (hunk 1 above)
Comment (alancast) [reply Contributing.md]: fixed!

**Pull request merged**